client/src/components/ViewProfile.js

```diff
@@ -24,11 +24,12 @@ const ViewProfile = () => {
   }, [profileId, delProfile, history]);
 
   return (
-    <Card color="black" textAlign="center" style={{ width: "auto" }}>
+    <Card color="black" style={{ width: "auto" }}>
       <Card.Content>
         <Card.Header>
           {`${profile.first_name} ${profile.last_name}`}
         </Card.Header>
+        <Card.Meta> {`${profile.groups}`} </Card.Meta>
```

Comment:
```suggestion
        <Card.Meta>{profile.groups.join(', ')}</Card.Meta>
```
Since `groups` is an array, we could join the list into a single string using `join` with `, ` in between each group. This assumes that `groups` will always be at least an empty array and not null